index.html

```diff
@@ -1,132 +1,178 @@
+                        guitars, steel guitars (such as pedal steels and dobros), banjos, and fiddles as well as
+                        harmonicas.</p>
                 </div>
 
                 <div class="wrap">
-                    <i class=""></i>
-                    <h3>Hip-hop</h3>
-                    <p>The saxophonist will play together with the synphony and many more instrumentds on the day one of the festival</p>
+                    <i class="fa-brands fa-r&b"></i>
+                    <h3>R&B</h3>
+                    <p>The term "rhythm and blues," often called "R&B," originated in the 1940s when it replaced "race
+                        music" as a general marketing term for all African American music, though it usually referred
+                        only to secular, not religious music.</p>
                 </div>
 
                 <div class="wrap">
-                    <i class=""></i>
-                    <h3>POP</h3>
-                    <p>The saxophonist will play together with the synphony and many more instrumentds on the day one of the festival</p>
+                    <i class="fa-solid fa-Hip-hop"></i>
+                    <h3>Hip-hop</h3>
+                    <p>Hip-hop is a genre of music most often characterized by a strong, rhythmic beat and a rapping
+                        vocal track. The genre originated in New York City in the 1970s as a cultural exchange among
+                        Black, Latino, and Caribbean youth and has grown into one of the most consumed genres of music
+                        in the United States</p>
                 </div>
-
                 <div class="wrap">
-                    <i class=""></i>
-                    <h3>Jaz</h3>
-                    <p>The saxophonist will play together with the synphony and many more instrumentds on the day one of the festival</p>
+                    <i class="fa-solid fa-kizomba"></i>
+                    <h3>Kizomba</h3>
+                    <p>Kizomba music emerged as a fusion of Semba, Angolan Merengue, Kilapanga, and further Angolan
+                        music influences: It slowed down the cadence of songs and added a stronger bass line to the
+                        composition of instruments.</p>
                 </div>
 
             </section>
+            <a class="link-program">SEE THE WHOLE PROGRAM</a>
+            <button class="button">Come to have a great time with us</button>
         </section>
-        <section class="Artists">
-            <h3>Featured Artist</h3>
+
+        <section class="speakers">
+            <h3>Featured Artists</h3>
             <hr>
-            <div class="Artist-list"></div>
+
+            <div class="speaker-wrap"></div> 
+
+            <div class="speakers-elt">
+            
+            </div>
+            <!-- <div class="more">MORE <img src="style/images/arrow-down.png" alt=""></div> -->
         </section>
```

Comment:
[OPTIONAL] Please kindly get rid of commented code for it makes your code base look untidy and messy.